go/vt/worker/grpcvtworkerclient/client.go

```diff
@@ -36,11 +36,11 @@ import (
 )
 
 var (
-	cert = flag.String("vtworker_client_grpc_cert", "", "the cert to use to connect")
-	key  = flag.String("vtworker_client_grpc_key", "", "the key to use to connect")
-	ca   = flag.String("vtworker_client_grpc_ca", "", "the server ca to use to validate servers when connecting")
-	crl  = flag.String("vtworker_client_grpc_crl", "", "the server crl to use to validate server certificates when connecting")
-	name = flag.String("vtworker_client_grpc_server_name", "", "the server name to use to validate server certificate")
+	cert = flag.String("vtworker_client_grpc_cert", "", "(DEPRECATED) the cert to use to connect")
```

Comment:
Can we add a deprecation warning when the vtworker binary starts up?

Reply:
Sure. I'm assuming you mean print it to STDERR/console. I'll add something.

Reply:
```
$ vtworker
*** This is a legacy sharding tool that will soon be removed! Please use VReplication instead https://vitess.io/docs/reference/vreplication/ ***
F0513 00:23:34.072379   42823 server.go:224] topo_global_server_address must be configured
```

Reply:
`Please use VReplication instead` -> `Please use MoveTables or Reshard commands instead`
WDYT?

Reply:
Yeah, if that makes more sense to you — being somebody who is familiar with the legacy sharding stuff  — then I'm all for it. I'll make this change shortly. Thanks!